cmd/minikube/cmd/start.go

```diff
@@ -161,7 +161,7 @@ func runStart(cmd *cobra.Command, args []string) {
 	starter, err := provisionWithDriver(cmd, ds, existing)
 	if err != nil {
 		node.MaybeExitWithAdvice(err)
-		machine.MaybeDisplayAdvice(err, viper.GetString("driver"))
+		machine.MaybeDisplayAdvice(err, existing.Driver)
```

Comment:
this should be ds.Name, existing can be nil

Reply:
good call, thanks